# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wine/powerpc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fixes compilation on PowerPC - I do not yet know if the resulting binaries
work, too.

  - Rene Rebe <rene@exactcode.de>

diff -ur wine-20040505/tools/winebuild/import.c wine-20040505-fixed/tools/winebuild/import.c
--- wine-20040505/tools/winebuild/import.c	2004-04-08 06:41:14.000000000 +0200
+++ wine-20040505-fixed/tools/winebuild/import.c	2004-06-04 21:33:53.207386280 +0200
@@ -121,7 +121,7 @@
                                           "r16","r17","r18","r19","r20","r21","r22","r23",
                                           "r24","r25","r26","r27","r28","r29","r30","r31" };
 # else  /* __APPLE__ */
-# define ppc_high(mem) "(" mem ")@hi"
+# define ppc_high(mem) "(" mem ")@h"
 # define ppc_low(mem)  "(" mem ")@l"
 static const char * const ppc_reg[32] = { "0", "1", "2", "3", "4", "5", "6", "7",
                                           "8", "9", "10","11","12","13","14","15",
